src/test/ui/print_type_sizes/packed.rs

```diff
@@ -1,5 +1,6 @@
 // compile-flags: -Z print-type-sizes
 // compile-pass
+// ignore-force
```

Comment:
It'd be great I think to make a short note why we're ignoring force. This ability to ignore a force flag also feels pretty weird/wrong, at least highly unusual.

Reply:
Yes, there should be very few such cases, so each one can get a comment.